### PR TITLE
⚰️ Suppression de l'état projets dans utilisateur.aggregate (géré dans accès)

### PR DESCRIPTION
**Pull request by HubM**
Lié à l'issue https://linear.app/startup-potentiel/issue/POT-3102/nom-du-projet-manquant-dans-lemail-suite-au-retrait-puis-repartage-dun

### CODE REVIEW

**Review by HanaeY — DISMISSED** [review]

File: packages/specifications/src/projet/accès/stepDefinitions/accès.when.ts [hunk 1]
```diff
@@ -139,13 +139,40 @@ Quand(
   },
 );
 
+type AjouterAccèsProjetProps = {
+  identifiantProjet: string;
+  identifiantUtilisateur: string;
+  ajoutéPar?: string;
+  raison: Accès.AutoriserAccèsProjetUseCase['data']['raison'];
+};
+export async function ajouterAccèsProjet(
```
Comment (VioMrqs) [inline packages/specifications/src/projet/accès/stepDefinitions/accès.when.ts]: Pas utilisé

File: packages/specifications/src/utilisateur/inviterPorteur.feature [hunk 2]
```diff
@@ -18,6 +18,16 @@ Fonctionnalité: Inviter un porteur sur un projet
             | lauréat |
             | éliminé |
 
+
+    Scénario: Le mail d'invitation liste le projet après un retrait d'accès puis un ré-ajout
```
Comment (VioMrqs) [inline packages/specifications/src/utilisateur/inviterPorteur.feature]: Je trouve ce test beaucoup trop spécifique, c'est vraiment un edge case. A mon avis ce qu'on doit tester c'est plutôt
```
        Et le porteur a accès au projet <Statut>
        Et la liste des porteurs du projet <Statut> est mise à jour
```
Quand il y a invitation, retrait de l'accès puis ré-invitation
Le mail est lié donc on s'en fiche un peu IMO
